packages/shared-fixtures/fixtures/typescript/expressions/satisfies-expression.src.ts

```diff
@@ -0,0 +1,10 @@
+x satisfies T;
+x < y satisfies boolean; // (x < y) satisfies boolean;
+x === 1 satisfies number; // x === (1 satisfies number);
+x satisfies any satisfies T;
+
+const t2 = { a: 1, b: 1 } satisfies I1; // Error
+const t3 = { } satisfies I1; // Error
+const t4: T1 = { a: "a" } satisfies T1; // Ok
+const t5 = (m => m.substring(0)) satisfies T2; // Ok
+const t6 = [1, 2] satisfies [number, number];
```

Comment:
Just making sure inline function types are supported too...

```suggestion
const t6 = [1, 2] satisfies [number, number];
const t7 = (async (context) => context.value) satisfies (context: { value: string }) => Promise<string>;
```